Kernel/FormulaTransformer.cpp

```diff
@@ -438,92 +438,5 @@ void ScanAndApplyFormulaUnitTransformer::updateModifiedProblem(Problem& prb)
   prb.invalidateEverything();
 }
 
-
```

Comment:
There actually was a bug here related to alias analysis and a pure-virtual call (which incidentally was _really_ hard to find, thank you GCC), but luckily it's dead code, remove it.